test/e2e/utils.go

```diff
@@ -353,6 +354,21 @@ func waitForDeploymentStatusConditions(ctx context.Context, cl client.Client, ti
 	})
 }
 
+func waitForDaemonsetUpToDate(ctx context.Context, cl client.Client, timeout time.Duration, name, ns string, numberUpToDateAndAvailable int32) error {
```

Comment:
s/Daemonset/DaemonSet/

Maybe drop `UpToDate` from the name and use godocs to explain.